src/main/scala/rps/Game.scala

```diff
@@ -4,24 +4,40 @@ import Move._
 import Result._
 import io.buildo.enumero.{CaseEnumIndex, CaseEnumSerialization}
 
-object Game {
-  def play(userMove: Move): Response = {
-    /* Ask for user input */
+import scala.concurrent.Future
+import wiro.annotation._
+
+// API definition
+
+@path("rps")
+trait GameApi {
+  @command
+  def play(
+      userMove: Move
+  ): Future[Either[Throwable, Response]]
+}
+
+// API implementation
+class GameApiImpl() extends GameApi {
+  import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
This works, but it's better to define a single `ExecutionContext` to use across all controllers, try passing the execution context to `GameApiImpl` as a parameter.